app/pages/lab/visibility.cjsx

```diff
@@ -229,6 +240,18 @@ module.exports = React.createClass
             </tbody>
           </table>
         </div>}
+
+      {if @state.workflows.length > 0
+        meta = @state.workflows[0].getMeta()
+
+        <Paginator
+          className="lab-visibility-paginator"
```

Comment:
If you're ok with waiting, #3908 is close to merging and I'll get them in with that and you can rebase after that is merged.